documentation/book/ref-sample-kafka-resource-config.adoc

```diff
@@ -0,0 +1,189 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-producer-configuration-{context}']
+= Sample Kafka YAML configuration
+
+For help in understanding the configuration options available for your Kafka deployment, refer to sample YAML file provided here.
+
+[source,shell,subs="+quotes,attributes"]
+
+----
+apiVersion: kafka.strimzi.io/v1beta1
```

Comment:
Currently this YAML is too long and causes the reader to scroll up and down to refer between the YAML structure and the callout text. There are a couple of ways we could fix that (which aren't mutually exclusive):

* Split out the kafka,zookeeper and remaining parts
* Elide the full configuration (e.g. commented ellipsis ` # ...` in the metrics) and then give the full configuration of that property in the linked-to documentation.

Reply:
I tried splitting into sections. I think we lost more than we gained. It became a little incoherent. But I've dropped the metrics definitions, which has shrunk it a lot. When we have the metrics PR merged, I'll add that content 'over there'.